tox.ini

```diff
@@ -8,13 +8,6 @@ envlist =
        {py27,py34,py35}-django{19}
 
 [testenv]
-basepython =
-            py27: python2.7
-            py32: python3.2
-            py33: python3.3
-            py34: python3.4
-            py35: python3.5
-
```

Comment:
I have it in a separate commit, and have some commentary on why I did it in the commit message.